llama-index-integrations/tools/llama-index-tools-jina/llama_index/tools/jina/base.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from yarl import URL
+from typing import Dict, List
+from llama_index.core.schema import Document
+from llama_index.core.tools.tool_spec.base import BaseToolSpec
+
+JINA_SEARCH_URL_ENDPOINT = "https://s.jina.ai/"
+
+
+class JinaToolSpec(BaseToolSpec):
```

Comment:
oh neat, this doesn't require an API key?

Reply:
Where i mentioned that it requires API key?

Reply:
You didn't -- I was just surprised that it a service like this works without one